### ISSUE TITLE
Feature discussion: Pagination

### ISSUE THREAD

**Issue by frozenpandaman** [Rotonde/rotonde-client#70]
Moving discussion from #68 here:

>Once a timeline "fills up" with more recent stuff (specifically, 42 entries + the $rotonde "bot" post) there's no way to see older stuff. Should we allow for multiple pages or even pagination/infinite scrolling so you can see all the way back?

For whispers/mentions, you can use `filter` to see missed/older posts, but I don't think there's a way to see older (or not, depending on # of users you're following) posts from other users. Thus, your feed/homepage is essentially limited to the 42 most recent posts (including your own).

**Comment by eelfroth** [comment]
it would be nice to have the ability to filter by timestamp i.e. `filter 3 days ago` or `filter 19.10.2017` or `filter 1508520410247`.
if we did this, it would also be a way to do pagination. a "load more" button in the bottom could fill in the timestamp of the last displayed post into the filter, making the feed only show posts that came before it.

**Comment by neauoire** [comment]
I second this! 👍

**Comment by neauoire** [comment]
DONE :D
